oat-sa/tao-item-runner-qti-fe
# Backport/BSA-391/Allow to disable figure widgets

### Related to: https://oat-sa.atlassian.net/browse/BSA-391

Backport of https://github.com/oat-sa/tao-item-runner-qti-fe/pull/389 to version November 2023 LTS

Requires: 

### Summary

Manage a feature flag for disabling the figure widget.

### Details

Some customers rely on the block positioning of the figures for aligning images in columns. The figure widget sets them to inline positioning, hence the need to have a feature flag for disabling it. The other possibility is to roll back the feature, but this is less handy.

The feature can be disabled thanks to the feature flag `FEATURE_FLAG_DISABLE_FIGURE_WIDGET`.
The feature is enabled by default

### How to test

- have all dependencies (other PR) installed
- edit an item and place images on the canvas. Check the attached ticket for a sample item.
- turn the feature flag on and off, and see the differences

It can also be tested on the following temporary environments:
- figure disabled: http://test-bsa-391-disable-figure.playground.kitchen.it.taocloud.org:44661/
- figure allowed: http://test-bsa-391-enable-figure.playground.kitchen.it.taocloud.org:43571/
